robocop/checkers/comments.py

```diff
@@ -150,19 +170,12 @@ def check_invalid_comments(self, name, node):
 
     def check_comment_content(self, token, content):
         content = content.lower()
-        if "todo" in content:
-            self.report(
-                "todo-in-comment",
-                todo_or_fixme="TODO",
-                lineno=token.lineno,
-                col=token.col_offset + 1 + content.find("todo"),
-            )
-        if "fixme" in content:
+        for marker in [term for term in self.todos if term.lower() in content]:
             self.report(
                 "todo-in-comment",
-                todo_or_fixme="FIXME",
+                marker=marker.upper(),
```

Comment:
Should we always display it with upper()? Maybe we could define markers as we want them to display, ie:

```
todo-in-comment:todos:TODO,FIXME,Remove me,Fix this!
```
or display it like it's displayed in the source code ('Found 'fixme' or Found 'TODO"). Just thinking aloud what would be best approach here.

Reply:
Agree, that guidance should be accurate as possible... We need to preserve the original content and use that when fetching the violation.